client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -21,20 +21,46 @@ const ProfileInfoEdit = () => {
   const dispatch = useDispatch();
 
   const {
-    teamId, teamName, orgName, email,
+    teamId, teamName, orgName, email, profilePic,
   } = useSelector(
     (state) => state.team,
   );
 
-  const [profileData, setInputs] = useState({ teamName, orgName, email });
+  const [setProfileData, setInputs] = useState({
```

Comment:
I think what Akshay meant was changing `setInputs` to `setProfileData`, i.e.:
`const [ profileData, setProfileData] = useState ...`

Reply:
yep just change of naming to maintain consistency